pkg/kubectl/history.go

```diff
@@ -305,6 +305,32 @@ func applyHistory(ds *extensionsv1beta1.DaemonSet, history *appsv1beta1.Controll
 	return clone, nil
 }
 
+// controlledSSHistories returns all ControllerRevisions controlled by the given StatefulSet
+func controlledSSHistories(apps clientappsv1beta1.AppsV1beta1Interface, namespace, name string) (*appsv1beta1.StatefulSet, []*appsv1beta1.ControllerRevision, error) {
```

Comment:
`controlledSSHistories` (for statefulset) and `controlledHistories` (for daemonset) are similar and thus redundant. What is the best way to introduce a generic function covering both, especially how should the function return correct type?